adanet-co2-monitor.ino

```diff
@@ -68,6 +73,12 @@ typedef enum
 } Error;
 
 static int8_t tempUnits;
+// number of updates per day 60s * 60min * 24hrs = 86400
+static constexpr int32_t UPDATES_PER_DAY = (86400 / DISPLAY_WAIT);
+static constexpr int32_t UPDATES_PER_WEEK = UPDATES_PER_DAY * 7;
+RTC_DATA_ATTR static uint16_t co2HistoryFifo[UPDATES_PER_WEEK];
```

Comment:
Could initialize with `RTC_DATA_ATTR static uint16_t co2HistoryFifo[UPDATES_PER_WEEK] = {0};` (which would eliminate need for `co2HistorySize` I think?)

Reply:
Not sure I follow... Use value of zero to indicate we're reached unused portion of history?

Reply:
Right, then just iterate through the full list `computeCo2Max` (since `0` won't interfere with maximum calculation)?

Reply:
I like it! Done.